bot.py

```diff
@@ -16,6 +16,7 @@
 from cogs.say import Say
 from cogs.tiles import Tiles
 from cogs.wiki import Wiki
+from cogs.markov import Markov
```

Comment:
This could be alphabetized for consistency

Reply:
fixed in 3f4489